frontend/test/__support__/e2e/commands/api/composite/createNativeQuestionAndDashboard.js

```diff
@@ -0,0 +1,19 @@
+Cypress.Commands.add(
+  "createNativeQuestionAndDashboard",
+  ({ nativeQuestionDetails, dashboardName = "Custom dashboard" } = {}) => {
```

Comment:
Looks great, thanks @nemanjaglumac 
I would rename `nativeQuestionDetails` to `question` since the command name already describes that it is for native questions

Reply:
Thank you for the review, @alxnddr .
Sounds like a valid suggestion. We can do that.

The other custom command `createQuestionAndDashboard` uses `questionDetails`, so I wanted to avoid any possible confusion between the two (although it's highly unlikely we're ever going to use them both in the same test).

Most of the tests will define question (details) before this function, like this:
```javascript
const question = {
...
}

cy.createNativeQuestionAndDashboard({ question }).then(()=> { /* do something */ });
```

If this still looks ok, I'll go and make the proposed change.